src/OpenSpaceToolkit/Astrodynamics/Trajectory/State.cpp

```diff
@@ -230,9 +230,9 @@ State State::operator+(const State& aState) const
         throw ostk::core::error::runtime::Wrong("Frame");
     }
 
-    if (this->getSize() != aState.getSize())
+    if (this->coordinatesBrokerSPtr_ != aState.coordinatesBrokerSPtr_)
```

Comment:
So for now, this will also check that the coordinates are in the exact same order yet?

Reply:
correct